source/elements/oneMKL/source/domains/blas/asum.rst

```diff
@@ -98,16 +100,84 @@ asum
       the real and imaginary parts of all elements of the vector).
 
 
+asum (USM Version)
+------------------
+
+.. container::
+
+   .. container:: section
+
+
+      .. rubric:: Syntax
+         :class: sectiontitle
+
+
+      .. container:: dlsyntaxpara
+
+
+         .. cpp:function::  sycl::event onemkl::blas::asum(sycl::queue &queue, std::int64_t n, const T *x, std::int64_t incx, T_res *result, const sycl::vector_class<sycl::event> &dependencies = {})
+   .. container:: section
+
+
+      .. rubric:: Input Parameters
+         :class: sectiontitle
+
+
+      queue
+         The queue where the routine should be executed.
+
+
+      n
+         Number of elements in vector ``x``.
+
+
+      x
+         Pointer to input vector ``x``. The array holding the vector
+         ``x`` must be of size at least (1 + (``n`` - 1)*abs(``incx``)).
+         See `Matrix and Vector
+         Storage <../matrix-storage.html>`__ for
+         more details.
+
+
+      incx
+         Stride of vector ``x``.
+
+
+      dependencies
+         List of events to wait for before starting computation, if any.
+         If omitted, defaults to no dependencies.
+
+
+   .. container:: section
+
+
+      .. rubric:: Output Parameters
+         :class: sectiontitle
+
+
+      result
+         Pointer to the output matrix where the scalar result is stored
+         (the sum of magnitudes of the real and imaginary parts of all
+         elements of the vector).
+
+
+   .. container:: section
+
+
+      .. rubric:: Return Values
+         :class: sectiontitle
+
+
+      Output event to wait on to ensure computation is complete.
+
+
 .. container:: familylinks
 
 
    .. container:: parentlink
 
 
       **Parent topic:** :ref:`blas-level-1-routines`
-      
-
-
-.. |image0| image:: ../equations/GUID-684BB993-83CA-4605-BD49-E493806C1ee1.png
+.. |image0| image:: ../equations/GUID-4F76F5A1-251F-4AC0-A2E0-A3B4B6F39ee1.png
```

Comment:
When I edit a doc with a GUID name image, I try to rename it to something more helpful.

Reply:
I will add it to our todo list